pkg/controller/backup_controller.go

```diff
@@ -951,8 +951,11 @@ func (c *backupController) deleteVolumeSnapshot(volumeSnapshots []*snapshotv1api
 			if vs.Status.BoundVolumeSnapshotContentName != nil &&
 				len(*vs.Status.BoundVolumeSnapshotContentName) > 0 {
 				vsc = vscMap[*vs.Status.BoundVolumeSnapshotContentName]
-				if vsc.Spec.DeletionPolicy == snapshotv1api.VolumeSnapshotContentDelete {
+				if nil != vsc && vsc.Spec.DeletionPolicy == snapshotv1api.VolumeSnapshotContentDelete {
```

Comment:
``` go
				if nil == vsc {
					logger.Errorf("Not find %s from the vscMap", vs.Status.BoundVolumeSnapshotContentName)
					return
				}
				if vsc.Spec.DeletionPolicy == snapshotv1api.VolumeSnapshotContentDelete {
					modifyVSCFlag = true
				}
```

Reply:
> Sorry, I didn't notice this in last review. It should be something like this.
> 
> ```go
> 				if nil == vsc {
> 					logger.Errorf("Not find %s from the vscMap", vs.Status.BoundVolumeSnapshotContentName)
> 					return
> 				}
> 				if vsc.Spec.DeletionPolicy == snapshotv1api.VolumeSnapshotContentDelete {
> 					modifyVSCFlag = true
> 				}
> ```

Yes，that's look like good, I have did that. Please check again. Thank you.